roles/confluent.test/molecule/kafka-connect-replicator-plain-kerberos-rhel/multi_rbac_converge.yml

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
this filename doesnt make sense

Reply:
RBAC will be added in the future!  And was originally present until the fallout from RBAC not being tested with Replicator hit the fan.